src/azure-cli-core/azure/cli/core/decorators.py

```diff
@@ -55,7 +55,7 @@ def hash256_result(func):
     @wraps(func)
     def _decorator(*args, **kwargs):
         val = func(*args, **kwargs)
-        if not val:
+        if val is None:
             raise ValueError('Return value is None')
```

Comment:
Can you check all the places that the `hash256_result` decorator is used and see the impact of this change?

Reply:
All functions that use this decorator return strings, so there is no effect